tests/test_acasclient.py

```diff
@@ -3674,6 +3687,80 @@ def test_010_validate_sdf_with_empty_aliases(self):
         meta_lot = self.client.get_meta_lot('CMPD-0000012-001')
         parent = meta_lot["lot"]["saltForm"]["parent"]
         self.assertEquals(0, len(parent['parentAliases']))
+    
+    @requires_absent_basic_cmpd_reg_load
+    def test_011_max_auto_lot_number(self):
+        # Load a file with a high lot number above the max auto lot number
+        file_a = Path(__file__).resolve().parent.joinpath(
+            'test_acasclient', 'ibuprofen_big_lot_number.sdf')
+        project_code = self.global_project_code
+        mappings = [
+                {
+                    "dbProperty": "Lot Number",
+                    "defaultVal": None,
+                    "required": False,
+                    "sdfProperty": "Lot Number"
+                },
+                {
+                    "dbProperty": "Lot Chemist",
+                    "defaultVal": "bob",
+                    "required": True,
+                    "sdfProperty": None
+                },
+                {
+                    "dbProperty": "Project",
+                    "defaultVal": project_code,
+                    "required": True,
+                    "sdfProperty": "Project Code Name"
+                },
+                {
+                    "dbProperty": "Parent Stereo Category",
+                    "defaultVal": STEREO_CATEGORY,
+                    "required": True,
+                    "sdfProperty": None
+                }
+            ]
+        response = self.client.register_sdf(file_a, "bob", mappings, dry_run=False)
+        self.assertIn('New compounds: 1', response['summary'])
+        # Confirm it got lot 4000, and extract the parent number
+        registered = read_registered_csv(response)
```

Comment:
ah cool that's a nice one to have because there are lots of tests where I fetch all lots and sort them just to get the one I loaded.